src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -89,6 +110,12 @@ async function _getBestPriceQuote(params: PriceQuoteParams): Promise<PriceInform
     const token = priceQuotes[0].token
     // console.log('Aggregated amounts', aggregationFunction, amounts, amount)
 
+    const winningPrices = priceQuotes
+      .filter(quote => quote.amount === amount)
```

Comment:
Didn't get why the filter here

Reply:
So we know the winning amount, now we want to know who is the winning price (or winning prices, is there's more than one with the same price)

Before u say so, im not concern with trying to get the amount and winning prices in one round by using a reduce function. I like the simplicity on line 109 (using the lib native max/min function)

Reply:
lol, not saying anything =P

Reply:
😂